src/child.rs

```diff
@@ -6,16 +6,28 @@ use tokio::process;
 /// Representation of a running or exited remote child process.
 ///
 /// This structure is used to represent and manage remote child processes. A remote child process
-/// is created via the [`Command`] struct through [`Session::command`], which configures the
+/// is created via the [`Command`](crate::Command) struct through [`Session::command`], which configures the
 /// spawning process and can itself be constructed using a builder-style interface.
 ///
+/// Calling [`wait`](RemoteChild::wait) (or other functions that wrap around it) will make the
+/// parent process wait until the child has actually exited before continuing.
+///
 /// Unlike [`std::process::Child`], `RemoteChild` *does* implement [`Drop`], and will terminate the
 /// local `ssh` process corresponding to the remote process when it goes out of scope. Note that
 /// this does _not_ terminate the remote process. If you want to do that, you will need to kill it
 /// yourself by executing a remote command like `pkill` to kill it on the remote side.
 ///
-/// Calling [`wait`](RemoteChild::wait) (or other functions that wrap around it) will make the
-/// parent process wait until the child has actually exited before continuing.
+/// As a result, `RemoteChild` cannot expose `stdin`, `stdout`, and `stderr` as fields for
+/// split-borrows like [`std::process::Child`] does. Instead, it exposes
+/// [`stdin`](RemoteChild::stdin), [`stdout`](RemoteChild::stdout),
+/// and [`stderr`](RemoteChild::stderr).
+/// Callers can call `.take()` to get the same effect as a split borrow, to use multiple streams concurrently:
```

Comment:
You can probably just text wrap this in with the text above. No need for the explicit line-break. Or if you really wanted a line break, you need two I think to make it a paragraph break.